beta/src/pages/reference/usestate.md

```diff
@@ -92,13 +78,13 @@ function handleClick() {
 
 #### Returns {/*setstate-returns*/}
 
-`setState` does not have a return value.
+`set` functions do not have a return value.
 
 #### Pitfalls {/*pitfalls*/}
 
 * Calling `setState()` only updates the state for the next render. If you read the `state` variable after calling `setState()`, you will still get the value that was on the screen before your call.
```

Comment:
Should we not call it `setState()` here?

Reply:
🤷🏻‍♀️